packages/react/src/breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { forwardRef } from "react";
+
+import { Box, type BoxProps } from "../box";
+import { extractSprinkles } from "../sprinkles";
+import * as styles from "./Breadcrumb.css";
+import { useBreadcrumbItems } from "./useBreadcrumbItems";
+
+export type BreadcrumbProps = BoxProps<
+  "nav",
+  {
+    children: React.ReactNode;
+    maxItems?: number;
+    separator?: React.ReactNode;
+  }
+>;
+
+export const Breadcrumb = forwardRef<HTMLElement, BreadcrumbProps>(
+  (
+    { children, className, maxItems = Infinity, separator = ">", ...props },
+    ref,
+  ) => {
+    const { restProps, sprinkleProps } = extractSprinkles(props);
+
+    const visibleItems = useBreadcrumbItems({
```

Comment:
I think we can skip the need to use `hooks` for it, rather can divide the component with more flexibility. Like
1. Breadcrumb
2. BreadcrumbList
3. BreadcrumbItem
4. BreadcrumbSeparator
5. BreadcrumbEllipsis

something like this. 
https://ui.shadcn.com/docs/components/breadcrumb

c.c @radnan vai

Reply:
I think we can skip this and stick with the current implementation